ui/component/app/view.jsx

```diff
@@ -272,9 +279,17 @@ function App(props: Props) {
     // signInSyncPref is cleared after sharedState loop.
     if (readyForSync && hasVerifiedEmail) {
       // In case we are syncing.
-      syncSubscribe();
+      syncLoop();
+      console.log('triggered');
     }
-  }, [readyForSync, hasVerifiedEmail, syncSubscribe]);
+    // @if TARGET='web'
+    window.addEventListener('focus', () => syncLoop(true));
+
+    return () => {
+      window.removeEventListener('focus', () => syncLoop(true));
```

Comment:
You need to pull this into it's own function so the event listener is removed properly

```
window.removeEventListener('focus', doSyncOnFocus)
```